docs/release_docs/REAN_Platform_Deployment_Workflows.md

```diff
@@ -0,0 +1,112 @@
+# REAN Platform Deployment workflows
+
+## PR-CI-CD
+Mode of trigger: ```Automated```
+
+ PR Workflow is triggered automatically whenever a PR with source branch as a feature/* branch is created against the target branch as develop branch.
+ 
+ Release Process Workflow Diagram.
+ ![PR-ci-cd_workflow](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/pr-ci-cd_workflow.png?raw=true)
```

Comment:
The links are not working
![image](https://github.com/REAN-Foundation/reancare-service/assets/30074835/57e92265-e731-4c17-a124-e0bf86f5d1ad)
![image](https://github.com/REAN-Foundation/reancare-service/assets/30074835/be6e2396-0f97-4d5e-a5cd-485dfe32f9a2)

Please fix the links or add the diagrams directly on the page